package.json

```diff
@@ -73,7 +73,7 @@
     "update-fnm-package": "node ./.ci/prepare-fnm-package.js",
     "verify-fnm-package": "node ./.ci/prepare-fnm-package.js --fail-on-difference",
     "bootstrap": ".ci/bootstrap",
-    "test": "esy x TestFnm.exe",
+    "test": "bash -c '(esy x TestFnm.exe)'",
```

Comment:
What do you think about changing the test to use `--shell=bash` explicitly instead of implying it here?

Reply:
😂

That's what I did first, but I thought you would want to keep testing `System.Shell.infer()`. Is that tested elsewhere?

Reply:
I suppose it is tested in the feature tests since they don't include `--shell=bash` explicitly. Would you prefer `--shell=bash`?

Reply:
I think that yes. This is a smoke test, not a full-blown test. I'd rather unit-test it with a feature test, where lots of things are implicit